sanic_testing/testing.py

```diff
@@ -109,28 +108,20 @@ async def _local_request(self, method: str, url: str, *args, **kwargs):
                     do_recv = websocket.recv
 
                     async def send(data):
-                        ws_proxy.received.append(data)
+                        ws_proxy.sent.append(data)
                         await do_send(data)
 
                     async def recv():
                         message = await do_recv()
-                        ws_proxy.sent.append(message)
+                        ws_proxy.received.append(message)
 
                     websocket.send = send  # type: ignore
                     websocket.recv = recv  # type: ignore
 
-                    async def do_mimic():
-                        try:
-                            await mimic(websocket)
-                        except ConnectionClosedOK:
-                            ...
-                        else:
-                            await websocket.send("")
-
-                    task = self.app.loop.create_task(do_mimic())
-
-                    while not task.done():
-                        await sleep(0.1)
+                    try:
+                        await mimic(websocket)
+                    except ConnectionClosedOK:
+                        pass
```

Comment:
We still want to add that last send if the connection is still open

Reply:
What's the purpose of that check? Can we instead do check `websocket.closed` Boolean value?

Reply:
Usually a blank message is sent to signify a connection should be closed. But, that should only happen if it is still open. A lot of people might add that to their `mimic`.

Reply:
Can we do `websocket.close()` for closing it? I think it is more explicitly by this way.

Reply:
The connection is already being closed by the context manager.